content/articles/encapsulation-in-c++/index.md

```diff
@@ -0,0 +1,132 @@
+#### Encapsulation in C++
+### Introduction to Encapsulation
+Encapsulation is the method of grouping data and features into a single part. This part is known as a **class**. It is done to protect straightforward access to the data. Encapsulation is one of the **Object-Oriented Programming (OOPs)** features.
+Construction of user-limited classes in **C++** supports encapsulation and data hiding. 
+### Table of Contents
```

Comment:
@jamarh35 You should add a space after each header. I previously asked you to remove extra space, but they can't have no space at all between the header and the sentences (do this with all headers).